packages/client/links/httpBatchLink/index.d.ts

```diff
@@ -1 +0,0 @@
-export * from '../../dist/./links/httpBatchLink';
```

Comment:
Unfortunately we need all these files that copy the structure of our `exports` because some bundlers (like React Native's Metro) do not support the `exports` field.